.github/workflows/tests.yml

```diff
@@ -109,6 +115,8 @@ jobs:
           GROQ_API_KEY: ${{ secrets.GROQ_API_KEY }}
           ASSEMBLYAI_API_KEY: ${{ secrets.ASSEMBLYAI_API_KEY }}
           FAL_KEY: ${{ secrets.FAL_KEY }}
+          PLAYAI_API_KEY: ${{ secrets.PLAYAI_API_KEY }}
```

Comment:
You'll need to use play.ht keys, not play.ai keys, with this SDK. This may be the reason for the failing auth in the tests.